src/Starcounter.VS.CSBoilerplateApplicationProjectTemplate/Api/MainHandlers.cs

```diff
@@ -0,0 +1,31 @@
+﻿using $safeprojectname$.ViewModels;
+using Starcounter;
+
+namespace $safeprojectname$.Api
+{
+    internal static class MainHandlers
+    {
+        public static void Register()
+        {
+            Application.Current.Use(new HtmlFromJsonProvider());
+            Application.Current.Use(new PartialToStandaloneHtmlProvider());
+
+            Handle.GET("/$safeprojectname$", () =>
+            {
+                var page = new HelloStarcounterPage
+                {
+                    Visitors = {Data = Db.SQL<GuestBookEntry>("SELECT g FROM GuestBookEntry g")}
+                };
+
+                if (Session.Current == null)
+                {
+                    Session.Current = new Session(SessionOptions.PatchVersioning);
+                }
+
+                page.Session = Session.Current;
+
+                return page;
```

Comment:
A few does to help developers get started like setting up a APS.NET MVC project. The thought here is to help newcomers get started with developing Starcounter apps with a simple example projects.

This could also be used internally to help keeping our apps consistent.